### PR TITLE
[3.10] bpo-46878: Purge 'non-standard' from sqlite3 docstrings (GH-31612)

### PR DESCRIPTION
**Pull request by erlend-aasland**
(cherry picked from commit 4d95fa1ac5d31ff450fb2f31b55ce1eb99d6efcb)

https://bugs.python.org/issue46878

Automerge-Triggered-By: GH:JelleZijlstra

### CODE REVIEW

**Comment by erlend-aasland** [comment]
This is a backport of GH-31612

File: Modules/_sqlite/connection.c [hunk 1]
```diff
@@ -1452,13 +1450,13 @@ _sqlite3.Connection.executemany as pysqlite_connection_executemany
     parameters: object
     /
 
-Repeatedly executes a SQL statement. Non-standard.
+Repeatedly executes an SQL statement.
```
Comment (JelleZijlstra) [inline Modules/_sqlite/connection.c:1453]: depends how you pronounce "SQL" :)

**Review by JelleZijlstra — APPROVED** [review]

File: Modules/_sqlite/connection.c (hunk 1 above)
Comment (erlend-aasland) [reply Modules/_sqlite/connection.c:1453]: Certainly not "sequel", I do hope! 😂

Comment (AlexWaygood) [reply Modules/_sqlite/connection.c:1453]: 🐿️

Comment (JelleZijlstra) [reply Modules/_sqlite/connection.c:1453]: I do actually say "sequel", I hope that won't get me fired

Comment (erlend-aasland) [reply Modules/_sqlite/connection.c:1453]: More seriously, here's how I think of it: 🙂 The official [^1] way of pronouncing SQLite is Ess Cue El Lite. In order to maintain some kind of consistency in the sqlite3 docs, I always think of SQL as Es Cue El. Both styles are fine with me though, as long as we are consistent 🤓

[^1]: according to Dr. Hipp, the creator of Ess Cue El Lite

Comment (erlend-aasland) [reply Modules/_sqlite/connection.c:1453]: BTW, while we're semi off-topic, here's a very nice interview with Dr. Richard Hipp: https://corecursive.com/066-sqlite-with-richard-hipp/

Comment (JelleZijlstra) [reply Modules/_sqlite/connection.c:1453]: Thanks, that was an interesting read!

**Pull request merged**